flow/workflows/cdc_flow.go

```diff
@@ -226,6 +226,14 @@ func CDCFlowWorkflow(
 	logger := log.With(workflow.GetLogger(ctx), slog.String(string(shared.FlowNameKey), cfg.FlowJobName))
 	flowSignalChan := model.FlowSignal.GetSignalChannel(ctx)
 
+	// ugly, find a better way to represent this later
+	if cfg.DisablePeerdbColumns {
```

Comment:
this seems like a ui flag instead of a config flag. @iskakaushik was thinking of splitting frontend vs backend protos, which this would benefit from